app/views/users/show.html.erb

```diff
@@ -14,15 +14,23 @@
         <br>
         <p> <strong> Total Downvotes: </strong> <%= downvote_count(@user)  %> </p>
         <br>
+        <svg class="graph" id=<%= "graph#{@user.id}" %> ></svg>
     </div>
     <div class="col-md-6">
+    <h2>Beauty is in the eye of the beholder...</h2>
   <% @places.each do |place| %>
-    <div class="votebox">
+
+    <div class="votebox vote" data-upvotes=<%= place.upvotes.where({user: @user}).count %> data-downvotes=<%= place.downvotes.where({user: @user}).count %> data-vote-percentage=<%= calculate_upvote_percentage( place.upvotes.where({user: @user}).count, place.downvotes.where({user: @user}).count ) %>  id=<%= place.id %> >
```

Comment:
can this logic be moved into a helper so it's testable?  `place.upvotes.where...`